eventapi.py

```diff
@@ -0,0 +1,56 @@
+import json
+from datetime import datetime
+from flask import request
+from flask_restful import Resource, abort
+
+from event import Event
+from eventvalidations import *
+
+def read_args(request):
+    return json.loads(request.data.decode('utf-8'))
+
+class EventAPI(Resource):
+    def get(self, e_id):
+        event = Event.get_by_id(e_id)
```

Comment:
Where is the validations on the event_id being passed in ? .. never trust user input

Reply:
👍 I commented on the wrong line, this is good now